mmv1/templates/terraform/examples/database_migration_service_migration_job_postgres_to_postgres.tf.erb

```diff
@@ -0,0 +1,91 @@
+data "google_project" "project" {
+}
+
+resource "google_sql_database_instance" "source_csql" {
+  name             = "<%= ctx[:vars]["source_csql"] %>"
+  database_version = "POSTGRES_15"
+  settings {
+    tier = "db-custom-2-13312"
+    deletion_protection_enabled = false
+  }
+  deletion_protection = false
+}
+
+resource "google_sql_ssl_cert" "source_sql_client_cert" {
+  common_name = "<%= ctx[:vars]["source_sqldb_cert"] %>"
+  instance    = google_sql_database_instance.source_csql.name
+
+  depends_on = [google_sql_database_instance.source_csql]
+}
+
+resource "google_sql_user" "source_sqldb_user" {
+  name     = "<%= ctx[:vars]["source_sqldb_user"] %>"
+  instance = google_sql_database_instance.source_csql.name
+  password = "<%= ctx[:vars]["source_sqldb_pass"] %>"
+
+  depends_on = [google_sql_ssl_cert.source_sql_client_cert]
+}
+
+resource "google_database_migration_service_connection_profile" "source_cp" {
+  location              = "us-central1"
+  connection_profile_id = "<%= ctx[:vars]["source_cp"] %>"
+  display_name          = "<%= ctx[:vars]["source_cp"] %>_display"
+  labels = {
+    foo = "bar"
+  }
+  postgresql {
+    host     = google_sql_database_instance.source_csql.ip_address.0.ip_address
+    port     = 3306
+    username = google_sql_user.source_sqldb_user.name
+    password = google_sql_user.source_sqldb_user.password
+    ssl {
+      client_key         = google_sql_ssl_cert.source_sql_client_cert.private_key
+      client_certificate = google_sql_ssl_cert.source_sql_client_cert.cert
+      ca_certificate     = google_sql_ssl_cert.source_sql_client_cert.server_ca_cert
+    }
+    cloud_sql_id = "<%= ctx[:vars]["source_csql"] %>"
+  }
+
+  depends_on = [google_sql_user.source_sqldb_user]
+}
+
+resource "google_sql_database_instance" "destination_csql" {
+  name             = "<%= ctx[:vars]["destination_csql"] %>"
+  database_version = "POSTGRES_15"
+  settings {
+    tier = "db-custom-2-13312"
+    deletion_protection_enabled = false
+  }
+  deletion_protection = false
+}
+
+resource "google_database_migration_service_connection_profile" "destination_cp" {
+  location              = "us-central1"
+  connection_profile_id = "<%= ctx[:vars]["destination_cp"] %>"
+  display_name          = "<%= ctx[:vars]["destination_cp"] %>_display"
+  labels = {
+    foo = "bar"
+  }
+  postgresql {
+    cloud_sql_id = "<%= ctx[:vars]["destination_csql"] %>"
+  }
+  depends_on = [google_sql_database_instance.destination_csql]
+}
+
+resource "google_database_migration_service_migration_job" "<%= ctx[:primary_resource_id] %>" {
+  location              = "us-central1"
+  migration_job_id = "<%= ctx[:vars]["migration_id"] %>"
+  display_name = "<%= ctx[:vars]["migration_id"] %>_display"
+  labels = {
+    foo = "bar"
+  }
+  static_ip_connectivity {
+  }
+  source             = "projects/${data.google_project.project.project_id}/locations/us-central1/connectionProfiles/<%= ctx[:vars]["source_cp"] %>"
```

Comment:
Here + other connection profile URLs, can we use a reference to their `id` field? That would also allow removing the `depends_on`.

Reply:
Done.